.gitignore

```diff
@@ -1,80 +1,361 @@
-#OS junk files
```

Comment:
Why did this file change so much?

Reply:
I have no clue I'll revert it since it did not have to change